tests/test_server.py

```diff
@@ -200,8 +202,21 @@ def send_request() -> None:
             Path("data") / "nlu.yml",
         ]:
             full_path = project_path / file
-            payload += full_path.read_text()
+            content = full_path.read_text()
```

Comment:
should this rather read the content as a dict, combine the dicts, and then dump that dict to yaml as a payload?

Reply:
I think both is fine. Reading and concatenating as dict is cleaner, but then we'd have to dump to file and read it again from file. Happy to adapt if you think it's crucial.

Reply:
I'm also happy to do this in a separate PR so we can unblock `master` asap.

Reply:
it seems a bit more future-proof to do it using Tom's suggestion. Let's merge this fix and do a follow-up PR?

Reply:
will do

Reply:
https://github.com/RasaHQ/rasa/pull/6595